java/src/apps/ActionListCoreBundle_da.properties

```diff
@@ -17,7 +17,7 @@ jmri.jmrit.throttle.ThrottleCreationAction = \u00c5bn k\u00f8rekontroller
 jmri.jmrit.MemoryFrameAction = \u00c5bn hukommelsesoverv\u00e5gning
 
 jmri.jmrix.loconet.locormi.LnMessageServerAction =  Start LocoNet server
-jmri.jmrit.withrottle.WiThrottleCreationAction =    Start WiThrottle
+jmri.jmrit.withrottle.WiThrottleCreationAction = Start WiThrottle server
```

Comment:
I don't know Danish, just copied from the LocoNet server line above. Apart from the language specifics, as a general GUI style, it probably should.